src/ui-kit/Select/useSelectPopup.tsx

```diff
@@ -0,0 +1,16 @@
+import { defaultEventBus } from '../../EventBus/defaultEventBus';
+import type { VNode } from '../../model/VNode';
+
+export const useSelectPopup = () => ({
```

Comment:
этот generic hook -  ui-kit/PopupManager/usePopupManager.ts

```suggestion
export const usePopupManager = <O extends Record<string, any>>() => ({
  open: (popup: VNode, options: O) =>
    defaultEventBus.fire('popup', { type: 'open', popup, ...options }),
  update: (popup: VNode, options: O) =>
    defaultEventBus.fire('popup', { type: 'update', popup, ...options }),
  close: () => defaultEventBus.fire('popup', { type: 'close' }),
});
```